packages/clay-upper-toolbar/src/index.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {ClayInput} from '@clayui/form';
+import classNames from 'classnames';
+import React from 'react';
+
+const UpperToolbarInput: React.FunctionComponent<
+	React.HTMLAttributes<HTMLInputElement>
+> = ({className, ...otherProps}) => (
+	<UpperToolbarItem expand>
+		<ClayInput.Group>
+			<ClayInput.GroupItem>
+				<ClayInput
+					className={classNames(className, 'form-control-inline')}
+					type="text"
+					{...otherProps}
+				/>
+			</ClayInput.GroupItem>
+		</ClayInput.Group>
+	</UpperToolbarItem>
+);
+
+const UpperToolbarItem: React.FunctionComponent<
```

Comment:
Hey @bryceosterhaus what do you think about removing the name of the main component in the other components, I find it a little reductive to put, `UpperToolbar` in front. Since this component is already in the context of the UpperToolbar and also in the package, I think this is understood to be a component within that context.